contracts/clients/src/index.ts

```diff
@@ -36,6 +36,14 @@ import { BlsWalletContracts, connectToContracts } from "./BlsWalletContracts";
 
 export * from "./signer";
 
+export {
+  default as SafeSingletonFactory,
+  SafeSingletonFactoryViewer,
+} from "./SafeSingletonFactory";
+
+export { default as AddressRegistryWrapper } from "./AddressRegistryWrapper";
+export { default as BlsPublicKeyRegistryWrapper } from "./BlsPublicKeyRegistryWrapper";
```

Comment:
Would be good to have tsdocs on these classes & their public functions & props

Reply:
https://github.com/web3well/bls-wallet/pull/556